co-simulation/patch/CarlaUE4.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# CarlaUE4.sh (or CarlaExecutor.sh, linked as CarlaUE4.sh)
+
+# Retrieve the Docker network name of the current host's container
+NETWORK=$(docker inspect --format '{{range $key, $value := .NetworkSettings.Networks}}{{$key}}{{end}}' $(hostname))
```

Comment:
Just to confirm my understanding: This should already exist because the docker compose from `./start-simulation` would have already created it?